job_spec/INSAR_ISCE_TEST.yml

```
@@ -37,6 +37,18 @@ INSAR_ISCE_TEST:
           - HRRR
           - HRES
           - GMAO
+    do_esd:
+      api_schema:
+        description: Whether or not to perform Enhanced Spectral Diversity for improved coregistration
+        defult: false
+        type: boolean
+    esd_coherence_threshold:
+      api_schema:
+        description: Coherence threshold for points to include when performing ESD
+        defult: 0.5
+        type: number
+        minimum: 0.0
+        maximum: 1.0
```

```suggestion
    esd_coherence_threshold:
      api_schema:
        description: Coherence threshold value, between 0 and 1, for points to include when performing ESD, or -1 for no ESD correction.
        defult: 0.5
        type: number
        minimum: -1.0
        maximum: 1.0
```
Technically, we'll accept values between -1 and 0 which will fail quickly as the container should raise a `ValueError` in that case. I don't think there's a way to say "either -1, or something between 0 and 1" here.